user-management/src/main/scala/com/normation/plugins/usermanagement/UserManagementService.scala

```diff
@@ -102,52 +133,48 @@ object UserManagementService {
 
   def computeRoleCoverage(roles: Set[Role], authzs: Set[AuthorizationType]): Option[Set[Role]] = {
 
-    def compareRights(r: Role): Option[Role] = {
-      if (r.name == "no_rights") {
+    def compareRights(r: Role, as: Set[AuthorizationType]): Option[Role] = {
+      if (r == Role.NoRights) {
```

Comment:
we have type, use them. We should not revert to string comparison appart for very good cases (check at system boudaries: parsing, serialisation, etc)